src/main/resources/application.yml

```diff
@@ -24,4 +24,4 @@ spring:
 # Status Callback configuration for Twilio. Port will be taken from server configuration
 hostconfig:
   host-address: localhost
-  protocol: http
\ No newline at end of file
+  protocol: http
```

Comment:
There is still a change in this file.

Reply:
Sure i did not look at this part lets have it as localhost